### PR TITLE
[RHOAIENG-5497] Enable the landing page

### PR DESCRIPTION
**Pull request by jeff-phillips-18**
Closes: [RHOAIENG-5497](https://issues.redhat.com/browse/RHOAIENG-5497)

## Description
Changes the default for `disableHome` to true in the dashboard config thus enabling the new Landing Page by default.

## How Has This Been Tested?
Tested locally

## Test Impact
Updated tests to verify that the default state shows the landing page.

## Request review criteria:
- [x] The developer has manually tested the changes and verified that the changes work
- [x] Commits have been squashed into descriptive, self-contained units of work (e.g. 'WIP' and 'Implements feedback' style messages have been removed)
- [x] Testing instructions have been added in the PR body (for PRs involving changes that are not immediately obvious).
- [x] The developer has added tests or explained why testing cannot be added (unit or cypress tests for related changes)

### CODE REVIEW

**Comment by andrewballantyne** [comment]
/hold

Needs a review from PM before we enable it for the product.

File: frontend/src/__tests__/cypress/cypress/pages/enabled.ts
Comment (andrewballantyne) [inline frontend/src/__tests__/cypress/cypress/pages/enabled.ts:3]: I don't understand why you changed this... the home page can still be disabled

Comment (jeff-phillips-18) [reply frontend/src/__tests__/cypress/cypress/pages/enabled.ts:3]: I could add it back but it's not set in any of the callers.

Comment (andrewballantyne) [reply frontend/src/__tests__/cypress/cypress/pages/enabled.ts:3]: You had a test to make sure the URL redirect was properly tested with and without the feature enabled. I'm feeling you had to have dropped something or we are missing something if it's no longer used.

This cannot be right I would think 🤔 Not sure where the gap is...

Comment (jeff-phillips-18) [reply frontend/src/__tests__/cypress/cypress/pages/enabled.ts:3]: I changed the test to verify that `/` takes you to the enabled page if the flag is disabled:
https://github.com/opendatahub-io/odh-dashboard/pull/2767/files#diff-34ec9e23c634748dc1ed7b69552b385bf9a11a5a746b7e4779c48e3302f9699aR19

**Review by andrewballantyne — APPROVED** [review]

**Pull request merged**
